### PR TITLE
Fix several storage related bugs

### PR DESCRIPTION
**Pull request by will-Do**
Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/pool/virsh_pool.py [hunk 1]
```diff
@@ -282,15 +282,18 @@ def check_pool_info(pool_info, check_point, value):
...
-        vol_capacity = "10000G"
-        vol_allocation = "10000G"
-        result = virsh.vol_create_as("oversize_vol", pool_name,
-                                     vol_capacity, vol_allocation, "raw")
-        check_exit_status(result, True)
-        new_pool_info = _pool.pool_info(pool_name)
-        check_pool_info(pool_info, "Capacity", new_pool_info['Capacity'])
-        check_pool_info(pool_info, "Allocation", new_pool_info['Allocation'])
-        check_pool_info(pool_info, "Available", new_pool_info['Available'])
+        # For NFS type pool, there's a bu(BZ#1077068)about allocate volume,
```
Comment (yumingfei) [inline libvirt/tests/src/virsh_cmd/pool/virsh_pool.py]: Is it being fixed? Maybe you can reserve the testcase and raise TestError with this bug-id.

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/pool/virsh_pool.py]: The bug might be fixed in 7.2, so we don't need run it for a long time

**Comment by PandaWei** [comment]
It looks OK for me.
ACK

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_create_from.py [hunk 2]
```diff
@@ -64,6 +71,12 @@ def run(test, params, env):
                 raise error.TestFail("No volume in pool: %s", src_pool_name)
         # Prepare vol xml file
         dest_vol_name = "dest_vol"
+        # According to BZ#1138523, we need inpect the right name
+        # (disk partition) for new volume
+        if dest_pool_type == "disk":
+            dest_vol_name = utlv.new_disk_vol_name(dest_pool_name)
```
Comment (Hao-Liu) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_create_from.py]: @will-Do I didn't find the implementation of `new_disk_vol_name`. Is there any PRs not submitted?

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_create_from.py]: Forgot to attach the dependent PR: https://github.com/autotest/virt-test/pull/2065
Thanks.

**Comment by will-Do** [comment]
Another dependent PR: https://github.com/autotest/virt-test/pull/2093

**Comment by will-Do** [comment]
Lots of bugs fixed in this PR, merge it as all tests run smoothly.

**Pull request merged**